CHANGELOG.md

```diff
@@ -1,1406 +1,2637 @@
+      - `autorange`
+      - `autorange_aperture_time_mode`
+      - `autorange_behavior`
+      - `autorange_minimum_aperture_time`
+      - `autorange_minimum_aperture_time_units`
+      - `autorange_minimum_current_range`
+      - `autorange_minimum_voltage_range`
+      - `autorange_threshold_mode`
+- Changed
+  - (Common) Fix [#1491](https://github.com/ni/nimi-python/issues/1491): import_attribute_configuration_buffer() fails intermittently when `list` or `array.array` is passed in.
+  - (Common) Update "Driver Version Tested Against", in documentation, with latest versions installed on nimi-bot. The version is 20.5.0 for NI-DCPower, NI-SWITCH, and NI-DMM. no changes on other drivers.
+
+#### [nidcpower] 1.3.1 - 2020-06-08
+- Changed
+  - (Common) Fix [#1473](https://github.com/ni/nimi-python/issues/1473): Unintentional dependency on pytest
+  - (Common) Fix [#1474](https://github.com/ni/nimi-python/issues/1474): Requires hightime>=0.2.0
+
+
+
+#### [nidcpower] 1.3.0 - 2020-05-21
+- Added
+  - API parity with NI-DCPower 20.0 by adding the following properties:
+      `Session.serial_number`
+      `Session.actual_power_allocation`
+      `Session.requested_power_allocation`
+      `Session.power_allocation_mode`
+- Changed
+  - (Common) Change the type of applicable properties and method parameters from `datetime.timedelta` to `hightime.timedelta` and from `datetime.datetime` to `hightime.datetime`. - [#744](https://github.com/ni/nimi-python/issues/744), [#1368](https://github.com/ni/nimi-python/issues/1368), [#1382](https://github.com/ni/nimi-python/issues/1382), [#1397](https://github.com/ni/nimi-python/issues/1397)
+  - (Common) Update "Driver Version Tested Against", in documentation, with latest versions installed on nimi-bot. The version is 20.0.0 for all modules except `nidigital`, for which it is 19.0.1.
+
+#### [nidcpower] 1.2.1 - 2020-04-21
+- Added
+  - (Common) Support for chained repeated capabilities. This allows things like
+      ``` python
+      session.sites[0, 1].pins['PinA', 'PinB'].ppmu_voltage_level = 4
+      ```
+
+      The repeated capabilities will be expanded to `'site0/PinA,site0/PinB,site1/PinA,site1/PinB'`
+
+#### [nidcpower] 1.2.0 - 2020-03-06
+- Added
+  - (Common) Zip file per driver for all examples and any helper files
+  - (Common) Link to zip file on examples documentation
+  - (Common) Support for Python 3.8
+  - `create_advanced_sequence()` - [#504](https://github.com/ni/nimi-python/issues/504)
+      Instead of a list of attribute IDs, you pass in a list of property names as strings
+      Includes example to see how to use it
+      Additional methods and properties that were made public (rather than private)
+      - `create_advanced_sequence_step()`
+      - `delete_advanced_sequence()`
+      - `active_advanced_sequence`
+      - `active_advanced_sequence_step`
```

Comment:
```suggestion
  - `create_advanced_sequence()` - [#504](https://github.com/ni/nimi-python/issues/504)
      - Instead of a list of attribute IDs, you pass in a list of property names as strings
      - Includes example to see how to use it
      - Additional methods and properties that were made public (rather than private)
        - `create_advanced_sequence_step()`
        - `delete_advanced_sequence()`
        - `active_advanced_sequence`
        - `active_advanced_sequence_step`
```